packages/veritone-react-common/src/components/ObjectDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import classNames from 'classnames';
+import { kebabCase } from 'lodash';
+
+import EngineOutputHeader from '../EngineOutputHeader';
+import PillButton from '../share-components/buttons/PillButton';
+import { msToReadableString } from '../../helpers/time';
+
+import styles from './styles.scss';
+
+const ObjectGroup = ({
+  objectGroup,
+  currentMediaPlayerTime,
+  onObjectClicked
+}) => {
+  let handleObjectClicked = (startTime, stopTime) => evt =>
+    onObjectClicked(startTime, stopTime);
+  return (
+    <span>
+      {objectGroup.series &&
+        objectGroup.series.map(objectData => {
+          return (
+            <PillButton
+              key={
+                'object-pill-' +
+                kebabCase(objectData.object.label) +
+                objectData.startTimeMs +
+                objectData.stopTimeMs
+              }
+              label={objectData.object.label}
+              info={
+                msToReadableString(objectData.startTimeMs) +
+                ' - ' +
+                msToReadableString(objectData.stopTimeMs)
+              }
+              className={styles.objectPill}
+              infoClassName={styles.objectAppearanceTime}
+              highlight={
+                currentMediaPlayerTime >= objectData.startTimeMs &&
+                currentMediaPlayerTime <= objectData.stopTimeMs
+              }
+              onClick={handleObjectClicked(
```

Comment:
read the comment I left on the fn, just disable the warning `// eslint-disable-line` and use an arrow fn `() => onObjectClicked(...)`